indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/pki/X509Certificate.kt

```diff
@@ -82,12 +81,12 @@ constructor(
 
             issuerUniqueID?.let {
                 +Asn1Primitive(
-                    1u.toImplicitTag(),
+                    1uL.toImplicitTag(),
                     Asn1BitString(it).let { byteArrayOf(it.numPaddingBits, *it.rawBytes) })
             }
             subjectUniqueID?.let {
                 +Asn1Primitive(
-                    1u.toImplicitTag(),
+                    1uL.toImplicitTag(),
```

Comment:
Couldn't we introduce this extension function and then keep the existing `1u`?

```kotlin
fun UInt.toImplicitTag(constructed: Boolean = false) =
    Asn1Element.Tag(this.toULong(), constructed = constructed, tagClass = TagClass.CONTEXT_SPECIFIC)
```

Reply:
this will go away after resolving #13 where i am still waiting for your feedback